apps/admin-dashboard/app/tailwind.css

```diff
@@ -16,3 +16,7 @@ button:focus-visible {
   outline-style: solid;
   outline-width: 1px;
 }
+
+.overflow-scroll {
+  overflow: auto;
+}
```

Comment:
Ahhhh, nice find on this!

I think I would prefer not overriding the default Tailwind `overflow-scroll` class cause it may cause some confusion in the future. Instead, what do you think about updating anywhere we use `overflow-scroll` to use `overflow-auto`?

I'm also thinking that to prevent us from accidentally using the `overflow-scroll` class in the future, we can add it to the Tailwind [blocklist](https://tailwindcss.com/docs/content-configuration#discarding-classes), ie:

```ts
// tailwind.config.js
module.exports = {
  blocklist: ['overflow-scroll']
}
```

Reply:
I can take a look and see if it functions the same. In the meantime though, here's a screenshot of what it looks like on Windows w/ the changes on both applications:
![image](https://github.com/colorstackorg/oyster/assets/112424267/cde0670b-a937-4261-823e-6d7314f4398b)
![image](https://github.com/colorstackorg/oyster/assets/112424267/76b85d7a-9872-47a1-bd46-b6096e8dcb94)